src/_locales/en_US/messages.json

```diff
@@ -23,6 +23,10 @@
         "message": "Currently Privacy Badger only checks if third parties are using cookies, HTML5 local storage, or canvas fingerprinting to track your browsing. Some of these domains may be using tracking methods that Privacy Badger can't detect.",
         "description": ""
     },
+    "options_enable_dnt_checkbox": {
+        "message": "Enable sending the <a href='https://www.eff.org/issues/do-not-track' target='_blank'>Do Not Track</a> signal",
```

Comment:
Sorry, now that I look at this again, should this be more to the point and say, "Send the Do Not Track signal"?

Reply:
Done